### ISSUE TITLE
error: module importing failed: ("'return' outside function", ('lldbinit.py', 91, None, 'return\n'))

### ISSUE THREAD

**Issue by fferri** [gdbinit/lldbinit#3]
trying to import lldbinit.py with

    command script import ~/Dev/lldbinit.py

gives the error:

```
error: module importing failed: ("'return' outside function", ('/Users/me/Dev/lldbinit.py', 91, None, 'return\n'))
  File "temp.py", line 1, in <module>
```

Version:

```
$ lldb --version
lldb-1000.11.37.1
  Swift-4.2
```

**Comment by MQuaresma** [comment]
Check my pull request for a fix: #2

**Comment by gdbinit** [comment]
Fixed!
